drivers/display/display_st7789v.c

```diff
@@ -206,7 +209,13 @@ static void *st7789v_get_framebuffer(const struct device *dev)
 static int st7789v_set_brightness(const struct device *dev,
 			   const uint8_t brightness)
 {
-	return -ENOTSUP;
+	const struct st7789v_config *config = dev->config;
+
+	if (config->backlight_dev) {
+		return led_set_brightness(config->backlight_dev, config->backlight_led, brightness);
```

Comment:
It seems to me that here board specific things are hacked into the driver. I am blocking it for now.

Reply:
@jfischer-no Is there a way to progress this patch? My goal is to have a backlight brightness interface that is not board specific. To me the `display_set_brightness` seems to be just that.